assets/multiqc_config.yml

```diff
@@ -1,11 +1,13 @@
 report_comment: >
   This report has been generated by the <a href="https://github.com/qbic-pipelines/rnadeseq" target="_blank">qbic-pipelines/rnadeseq</a>
   analysis pipeline. For information about how to interpret these results, please see the
-  <a href="https://github.com/qbic-pipelines/rnadeseq" target="_blank">documentation</a>.
+  <a href="https://github.com/qbic-pipelines/rnadeseq/blob/master/docs/output.md" target="_blank">documentation</a>.
 report_section_order:
-  software_versions:
+  "qbic-pipelines-rnadeseq-methods-description":
     order: -1000
```

Comment:
I think that, to prevent the linting error, this will have to be the last one. But it will now fail due to the links too. It's ok for me to leave it as it is, but have it in mind in case it throws an error in the future.

Reply:
Reordered it. I had added the `exists` key already, but somehow it got deleted again...

Many thanks for reviewing!